parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -368,7 +368,7 @@ renderTypeError e env src = case e of
   Mismatch {..} ->
     mconcat
       [ Pr.lines
-          [ "I found a value  of type:  " <> style Type1 (renderType' env foundLeaf),
```

Comment:
I think this extra space may have been intentional to allow the types and the `:`s to line up. I would move the space after the `:` to preserve that the types line up, or just revert this line altogether.

Reply:
I think this has caught me like 3 times now, I admit I don't find it very intuitive 🙃